app/types/feedback.ts

```diff
@@ -0,0 +1,4 @@
+export type TFeedback = {
```

Comment:
In typescript, it's not a commonly used convention to use `T` in front to label Type or `I` to label interface. That's usually for Java and C#. In this case the convention is to just do PascalCasing on the types. 

tl;dr: change from `TFeedback` to `Feedback`